### PR TITLE
Neo4j Backup and Restore Procedure with GKE Cronjob and GCS bucket example

### PR DESCRIPTION
**Pull request by mijaz18**
An example to backup a neo4j graph database deployed on Google Kubernetes Engine via Kubernetes Cronjob running on a custom schedule and offloading the backup to a Google Cloud Storage bucket. The example also includes a one off script that restores the database on a neo4j server.

### CODE REVIEW

File: examples/neo4j-backup-restore-via-gke-gcs-example/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+## Project Structure
+```
+.
+└── neo4j_backup_restore_via_gke_gcs_example
+ └── backup
+    └── deployment 
+          ├── backup-cronjob.yaml  #(Cronjob configuration)
+          └── deploy-exec.sh  #(Executable for backup deployment)
+    └── docker
+          ├── Dockerfile  #(Backup pod docker mage)
```
Comment (shakeebshams) [inline examples/neo4j-backup-restore-via-gke-gcs-example/README.md:14]:
```suggestion
          ├── Dockerfile  #(Backup pod docker image)
```

`image` i assume?

File: examples/neo4j-backup-restore-via-gke-gcs-example/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+$ ./restore/restore-exec.sh
+
+```
+
+## References
+
+This software uses the following open source packages:
+
+- [GKE Cronjobs](https://cloud.google.com/kubernetes-engine/docs/how-to/cronjobs)
+- [Artifact Registory](https://cloud.google.com/artifact-registry)
```
Comment (shakeebshams) [inline examples/neo4j-backup-restore-via-gke-gcs-example/README.md:107]:
```suggestion
- [Artifact Registry](https://cloud.google.com/artifact-registry)
```

File: examples/neo4j-backup-restore-via-gke-gcs-example/restore/download-backup.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#   BACKUP_NAME
+# Arguments:
+#   None
+#######################################
+
+export BACKUP_NAME=graph.db-backup
+
+echo "Fetch the latest copy of backup"
+response=$(gsutil ls -l gs://"<GCS_BUCKET>"|sort -k 2|tail -2|head -1 || true)
+latest_backup=$(${response[2]})
```
Comment (shakeebshams) [inline examples/neo4j-backup-restore-via-gke-gcs-example/restore/download-backup.sh:29]: Need to confirm that this is intended behavior given the nested execute statement `$()`.

Comment (mijaz18) [reply examples/neo4j-backup-restore-via-gke-gcs-example/restore/download-backup.sh:29]: Just confirmed this is the intended behavior

**Review by agold-rh — APPROVED** [review]

**Pull request merged**